server/controllers/finance/cash.js

```diff
@@ -51,6 +51,9 @@ exports.lookup = lookup;
 /** list all cash payment */
 exports.listPayment = listPayment;
 
+/** Checkin if the invoice is paid */
+exports.checkin = checkin;
```

Comment:
This could use a better name.  Maybe checkReversedPayment?  Or checkInvoicePayment?